apache-rat-core/src/test/java/org/apache/rat/analysis/matchers/CopyrightMatcherTest.java

```diff
@@ -53,91 +53,99 @@ public class CopyrightMatcherTest {
         for (String pfx : prefix) {
-            Arrays.stream((String[][]) source[4]).map(origin -> new String[] { pfx + origin[0], pfx + origin[1] })
-                    .forEach(arry::add);
-        }
-        result[4] = arry.toArray(new String[arry.size()][2]);
-        arry.clear();
-
-        for (String pfx : prefix) {
-            Arrays.stream((String[][]) source[5]).map(origin -> new String[] { pfx + origin[0], pfx + origin[1] })
-                    .forEach(arry::add);
+          Arrays.stream(args).map(origin -> new String[] { pfx + origin[0], pfx + origin[1] })
+                  .forEach(arry::add);
         }
-        result[5] = arry.toArray(new String[arry.size()][2]);
-        return result;
+        return arry.toArray(new String[arry.size()][2]);
     }
 
+            
+//    private static Arguments expandResults(String name, String startDate, String endDate, String owner,
+//            String[][] passAry, String[][] failAry) {
+//
+//        List<String[]> arry = new ArrayList<>();
+//        for (String pfx : prefix) {
+//            Arrays.stream(passAry).map(origin -> new String[] { pfx + origin[0], pfx + origin[1] })
+//                    .forEach(arry::add);
+//        }
+//        String[][] pass = arry.toArray(new String[arry.size()][2]);
+//        arry.clear();
+//
+//        for (String pfx : prefix) {
+//            Arrays.stream((String[][]) failAry).map(origin -> new String[] { pfx + origin[0], pfx + origin[1] })
+//                    .forEach(arry::add);
+//        }
+//        String[][] fail = arry.toArray(new String[arry.size()][2]);
+//
+//        return Arguments.of(name, startDate, endDate, owner, pass, fail);
+//    }
+
     private static void verify(String testName, String[][] pass, String[][] fail) {
-        assertEquals("Wrong number of pass/fail tests", TOTAL_TESTS, pass.length + fail.length);
+        assertEquals(TOTAL_TESTS, pass.length + fail.length, "Wrong number of pass/fail tests");
         Set<String> passSet = new HashSet<String>();
         Arrays.stream(pass).forEach(s -> passSet.add(s[0]));
         Set<String> failSet = new HashSet<String>();
         Arrays.stream(fail).forEach(s -> failSet.add(s[0]));
         for (String s : passSet) {
-            assertFalse(String.format("%s is in both pass and fail sets for %s", s, testName), failSet.contains(s));
+            assertFalse(failSet.contains(s), ()->String.format("%s is in both pass and fail sets for %s", s, testName));
         }
     }
 
-    public CopyrightMatcherTest(String testName, String start, String stop, String owner, String[][] pass,
+//    public CopyrightMatcherTest(String testName, String start, String stop, String owner, String[][] pass,
+//            String[][] fail) {
+//        verify(testName, pass, fail);
+//        matcher = new CopyrightMatcher(start, stop, owner);
+//        this.pass = pass;
+//        this.fail = fail;
+//        this.testName = testName;
+//    }
```

Comment:
pls remove old methods as commented-out code will never be removed again ... thx